src/components/DropDownButton/index.tsx

```diff
@@ -68,17 +68,23 @@ const Item = styled.div<{ border: boolean }>`
 	border-top: ${props =>
 		props.border ? `1px solid ${props.theme.colors.gray.main}` : '0'};
 	border-radius: ${props => px(props.theme.radius)};
+	background: white;
 
 	&:hover:enabled {
 		background: ${props => props.theme.colors.gray.light};
 	}
+
+	&:hover {
+		background: ${props => props.theme.colors.gray.light};
+		cursor: pointer;
+	}
 `;
 
 const JoinedButton = styled(Button)`
 	margin: 0;
 	width: 100%;
 `;
-
+// {..._.without('bg', props)}
```

Comment:
This line needs deleting rather than commenting out I think?

Reply:
agree, I'll clean this up